### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Kateryna199420**
- [DEMO LINK](https://<Kateryna199420>.github.io/layout_stars/)
- [TEST REPORT LINK](https://<Kateryna199420>.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -21,8 +21,8 @@ Follow
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_stars/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_stars/report/html_report/)
+- [DEMO LINK](https://<Kateryna199420>.github.io/layout_stars/)
+- [TEST REPORT LINK](https://<Kateryna199420>.github.io/layout_stars/report/html_report/)
```
Comment (Viktor-Kost) [inline readme.md:25]: And rewrite these links for Pull Request description please
```suggestion
- [DEMO LINK](https://Kateryna199420.github.io/layout_stars/)
- [TEST REPORT LINK](https://Kateryna199420.github.io/layout_stars/report/html_report/)
```

File: src/style.css [hunk 2]
```diff
@@ -1 +1,24 @@
...
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px; /* Adjust the size as needed */
+  height: 16px;
+  margin-right: 4px; /* Adjust the size as needed */
```
Comment (Viktor-Kost) [inline src/style.css:14]: 1. remove comments
2. last star shouldn't have margin

File: src/style.css [hunk 3]
```diff
@@ -1 +1,24 @@
...
+  background:url(./images/star.svg) center no-repeat;
```
Comment (Viktor-Kost) [inline src/style.css:15]:
```suggestion
  background: url(./images/star.svg) center no-repeat;
```

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]

**Review by nazarmatsevych — APPROVED** [review]
Awesome 👍

**Pull request closed without merging**